client/src/components/AssetBoard.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Grid } from '@mui/material';
+import { Asset } from 'models/Asset';
```

Comment:
1 - `Asset.ts` is used in `apiUtil.ts`, `CartList.tsx`, and its Redux counterparts. Since it was used in multiple places, we thought it would make sense to store it globally. That said, without `AssetBoard` there would be no need for an Asset model since all other modules are built around `AssetBoard`. Is this the argument for moving `Asset` to `AssetBoard?`

2 - I'll rename the directory, that's a really good point.

Reply:
yes